source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -322,6 +322,12 @@ Http::FilterHeadersStatus JsonTranscoderFilter::encodeHeaders(Http::HeaderMap& h
 
   response_headers_ = &headers;
 
+  // if the client connection was http/1 then Trailer headers are prohibited
+  // (unless the transfer encoding is chunked, which it isn't), remove them
+  if (encoder_callbacks_->requestInfo().protocol() < Http::Protocol::Http2) {
```

Comment:
use != for enums.

Reply:
I can make that change, but felt that this was more future proof against adding more values to the enum.